apps/snitch_core/priv/repo/seed/orders.ex

```diff
@@ -38,13 +38,13 @@ defmodule Snitch.Seed.Orders do
 
     digest = %{
       cart: [user_id: user.id],
-      address: [user_id: user.id, address_id: address.id],
-      payment: [user_id: user.id, address_id: address.id],
-      processing: [user_id: user.id, address_id: address.id],
-      shipping: [user_id: user.id, address_id: address.id],
-      shipped: [user_id: user.id, address_id: address.id],
-      cancelled: [user_id: user.id, address_id: address.id],
-      completed: [user_id: user.id, address_id: address.id]
+      address: [user_id: user.id, address: address],
```

Comment:
Are you sure this works? It shouldn't because the `address` here is _not a `map`_, but an `Address.t` struct.
The `cast_embed` expects a `map` right?

Reply:
I extracted the fields using `Map.from_struct(Struct) |> Map.delete(:__meta__)`. I think it is working fine with that.

Reply:
It doesn't look like that... :thinking: 
The `address` is [this](https://github.com/aviabird/snitch/pull/104/files/33097b135bbc663189d6f0e35c5db6d2f528a003#diff-2b18d261b0e89d39b464d0ad5df0403bR36), fetched from the address table.
Just to be sure, I ran `mix ecto.reset` and it failed right after variants were seeded.